src/lib.rs

```diff
@@ -226,7 +226,7 @@ impl<T: Send> ThreadLocal<T> {
     }
 
     #[cold]
-    fn insert(&self, data: T) -> &T {
+    fn insert(&self, thread: Thread, data: T) -> &T {
         let thread = thread_id::get();
```

Comment:
This argument seems to be overwritten in the first line of the function.

Reply:
fixed. looks like CI fails because of MSRV.